packages/eslint-config-eslint/default.yml

```diff
@@ -106,14 +102,12 @@ rules:
     no-new: "error"
     no-new-func: "error"
     no-new-object: "error"
-    no-new-require: "error"
     no-new-wrappers: "error"
     no-octal: "error"
     no-octal-escape: "error"
     no-param-reassign: "error"
-    no-path-concat: "error"
-    no-process-exit: "error"
     no-proto: "error"
+    no-process-exit: "off"
```

Comment:
This is added due to the bug described [here](https://github.com/mysticatea/eslint-plugin-node/pull/211). Do we want to go ahead and merge this or do we want to wait on the next `eslint-plugin-node` release?

Reply:
Because the update of recommended configs is a breaking change, I have not updated that yet (I think that it's not a problem because deprecation is not removal). I will do the next major release after the ESTree matter is completed.

Reply:
Got it, thanks. In that case, I think we should leave this in for now.